tests/TestCase/Model/SearchTraitTest.php

```diff
@@ -83,7 +83,7 @@ public function testFinder()
     public function testFindSearchException()
```

Comment:
This test can be dropped.

Reply:
I wasn't sure if we wanna keep the manually thrown exception or go with the argument error.
But I aggree that providing a search named param is required 👍🏻

Reply:
I went through all this trouble of having finder with typed args so that we can just rely on the language and not have to add checks myself :laughing:.